internal/bundle/lint.go

```diff
@@ -414,6 +417,20 @@ func lintFile(file string) (*Results, error) {
 				uid := check.Uid
 				updateAssignedQueries(check, assignedQueries, globalQueriesByUid)
 
+				checks[check.CodeId] = struct{}{}
+				if _, ok := dataQueries[check.CodeId]; ok {
```

Comment:
at this point in the code, `CodeId` isn't set. If we move this logic to outside of this function, the file context is missing...

If we want to implement this error, we will need to use this in combination with the ocmpiled bundle map. The issue is that the bundlemap indexes queries by MRN, but this struct has only UIDs. There is no easy way to match which query gets which mrn/codeid